src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -172,11 +173,19 @@ public TypeConverter GetOrCreateConverterInstance(Type innerType)
             //
             [typeof(Array)] = new IntrinsicTypeConverterData((type) => new ArrayConverter()),
             [typeof(ICollection)] = new IntrinsicTypeConverterData((type) => new CollectionConverter()),
-            [typeof(Enum)] = new IntrinsicTypeConverterData((type) => new EnumConverter(type), cacheConverterInstance: false),
+            [typeof(Enum)] = new IntrinsicTypeConverterData((type) => CreateEnumConverter(type), cacheConverterInstance: false),
             [s_intrinsicNullableKey] = new IntrinsicTypeConverterData((type) => new NullableConverter(type), cacheConverterInstance: false),
             [s_intrinsicReferenceKey] = new IntrinsicTypeConverterData((type) => new ReferenceConverter(type), cacheConverterInstance: false),
         });
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2067:UnrecognizedReflectionPattern",
+            Justification = "Trimmer does not trim enums")]
```

Comment:
Is it possible for this to change in future versions of the trimmer? If so what is the plan?

Reply:
I don't believe it is possible for the trimmer to change this behavior. We make the same justification elsewhere:

https://github.com/dotnet/runtime/blob/ecc354a61c909a1960af7e5a73eb9a0d255c564e/src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs#L5510-L5519